### PR TITLE
Replace Karma + Mocha with Jest

### PR DESCRIPTION
**Pull request by havardholvik**
Replace Karma + Mocha with Jest.

<details>
  <summary>📦 Published PR as canary version: <code>1.3.7--canary.30.6719332745.0</code></summary>
  <br />
  
  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @whereby/jslib-media@1.3.7--canary.30.6719332745.0
  # or 
  yarn add @whereby/jslib-media@1.3.7--canary.30.6719332745.0
  ```
</details>

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -3,14 +3,15 @@
     "env": {
         "browser": true,
         "es6": true,
-        "mocha": true
+        "jest/globals": true,
+        "node": true
```
Comment (havardholvik) [inline .eslintrc:7]: Added to silence some warnings in our config files

File: src/webrtc/rtcStatsService.js [hunk 2]
```diff
@@ -1,8 +1,3 @@
-/* globals process */
```
Comment (havardholvik) [inline src/webrtc/rtcStatsService.js:1]: Removed as the node eslint rule complained about `process` 🤷‍♂️

File: tests/webrtc/baseRtcManagerSpec.js [hunk 3]
```diff
@@ -98,29 +97,27 @@ export function test(createRtcManager) {
...
                 describe("SDP_OFFER", () => {
                     it("sets the remote description", async () => {
-                        const { pc } = await rtcManager._connect(clientId);
-
-                        // Run
+                        const session = await rtcManager._connect(clientId);
                         const validSdp = (getValidSdpString() + "\n").split("\n").join("\r\n");
                         const offer = { type: "offer", sdpU: validSdp };
 
+                        session.isOperationPending = false; // HACK
```
Comment (havardholvik) [inline tests/webrtc/baseRtcManagerSpec.js:117]: I would love some help figuring out why I had to revert to this hack to make the test pass

**Review by thyal — COMMENTED** [review]
Super nice clean up! The logs are shown in console when running the tests, which makes it difficult to read (there's a lot..). Maybe we should disable the `debug` logs for the tests? what do you think?

**Comment by havardholvik** [comment]
> Super nice clean up! The logs are shown in console when running the tests, which makes it difficult to read (there's a lot..). Maybe we should disable the `debug` logs for the tests? what do you think?

I also dont like this output. I was wondering if we should take these log tweaks on in the separate task we have for the SDK to prevent logs in console. Maybe we just need to make the logging configurable from the calling environment....?

**Review by thyal — APPROVED** [review]
🍰 Nice, looks great!

**Pull request merged**